src/dataflow/src/logging/timely.rs

```diff
@@ -287,25 +287,29 @@ pub fn construct<A: Allocate>(
 
         let operates = operates.as_collection();
 
+        // Feedback delay for log flushing. This should be large enough that there is not a tight
+        // feedback loop that prevents high-throughput work, but not so large that we leave volumes
+        // of logging data around that we do not need. Effectively, a throughput v memory trade-off.
+        let delay = std::time::Duration::from_nanos(10_000_000_000);
+
         // Accumulate the durations of each operator.
         let mut elapsed = duration
-            .map(|(op, t, d)| (op, t, d as isize))
-            .as_collection()
-            .count_total();
+            .map(|(op, t, d)| ((op, ()), t, d as isize))
```

Comment:
We should explain the rationale for why you want to keep the map around before the `thin_collection` call because that wasn't obvious to me at all

Reply:
Good point, and `op` is a bad name for `((op, worker), elapsed)`. :D

Reply:
Wait nvm that comment. Still a good point, but looking at the deleted code.